juniper_official/Protocols/Mpls/check-lsp-flap-count-oc.rule

```diff
@@ -0,0 +1,104 @@
+/*
+ * Detects MPLS LSP state changes and notifies when state change count 
+ * increases frequently.
+ * One inputs control detection
```

Comment:
Please change to one input controls detection.

Reply:
Changed to "one input controls detection"